user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -434,10 +441,6 @@
                 collapseID="teamBuildd"
                 >
               </app-live-workshop>
```

Comment:
After this point, the type of cards seems to change from the "show more/show less" type to the "click to learn more" type. I think that the latter looks better, and there should only be one type so it doesn't look so different in contrast.

Reply:
See: https://i.imgur.com/MUujkLq.jpg

Reply:
Yeah Ali made new ones that we're planning to use, we're not planning to use both. We're updating the workshops at the bottom as soon as education finalizes